books/views.py

```diff
@@ -1,3 +1,8 @@
 from django.shortcuts import render
 
-# Create your views here.
+from books.models import Book
+
+
+def book_list(request):
+    books = Book.objects.all()[:1_000]
```

Comment:
1 mln rows was a bit too ambitious, it will tremendously slow down the page, with 1k rows you can already see the effect